sciencebox/values.yaml

```diff
@@ -569,156 +567,156 @@ eos-instance-config:
-##         03-simpleForm: |
-##           # Remove this to enable the full form
-##           c.SwanSpawner.options_form = open('/srv/jupyterhub/jupyterhub_form.html').read()
-##     singleuser:
-##       cpu:
-##         limit: .5
-##         guarantee: .5
-##       extraEnv:
-##         # The sharing functionality won't work because we are deploying oCIS and not oc10
-##         # This will be replaced in the future with the CS3 APIs
-##         # SHARE_CBOX_API_DOMAIN: LOCAL_CBOX_URL__CHANGE_ME
-##         # SHARE_CBOX_API_BASE: /cernbox/swanapi/v1
-##         HELP_ENDPOINT: https://raw.githubusercontent.com/swan-cern/help/up2u/
-##         # FIXME Create a galery for ScienceBox? Or the normal one should redirect to different instances?
-##         # GALLERY_URL: https://swan-gallery.web.cern.ch/
-## 
-##     # TODO For some reason this is not working... We're still debugging
-##     prePuller:
-##       hook:
-##         enabled: false
-## 
-##     ingress:
-##       annotations:
-##         kubernetes.io/ingress.class: nginx
-##         nginx.ingress.kubernetes.io/ssl-redirect: "true"
-##         nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
-##         #nginx.ingress.kubernetes.io/rewrite-target: /
-##         #nginx.ingress.kubernetes.io/secure-backends: "true"
-##       hosts:
-##       # tls:
-##       #   - secretName: swan-tls-cert
-##       #     hosts:
-##       #       - test.mydomain
-##     custom:
-##       cvmfs:
-##         deployDaemonSet: *cvmfsDeployDS
-##         deployCsiDriver: *cvmfsDeployCSI
-##         useCsiDriver: *cvmfsUseCSI
-##         repositories: *cvmfsRepos
-##       eos:
-##         deployDaemonSet: *eosDeployDS
-##         deployCsiDriver: *eosDeployCSI
-##         useCsiDriver: *eosUseCSI
-## 
-## 
-##     # To decide if we keep this or we manage the tls secret differently
-##     # If we keep it, maybe should change the name from swan.secrets.ingress to smth else
-##     # swan:
-##     #   secrets:
-##     #     ingress:
-##     #       cert:
-##     #       key:
+#
+# SWAN
+#
+swan:
+  #
+  # CVMFS mounts
+  cvmfs:
+    deployDaemonSet: &cvmfsDeployDS true
+    deployCsiDriver: &cvmfsDeployCSI false
+    useCsiDriver: &cvmfsUseCSI false
+    repositories: &cvmfsRepos
+      - cvmfs-config.cern.ch
+      - sft.cern.ch
+      - sft-nightlies.cern.ch
+  #
+  # EOS Fusex mount
+  eos:
+    deployDaemonSet: &eosDeployDS true
+    deployCsiDriver: &eosDeployCSI false
+    useCsiDriver: &eosUseCSI false
+  fusex:
+    checkMgmOnline:
+      enabled: true
+      eosMgmUrlAuto: true
+    fusex:
+      keytab:
+        secret: *sssKeytabName
+      hostMountpoint: /var/eos
+      config:
+        oauth2: 1
+        eos_mgm_alias: "sciencebox-mgm.default.svc.cluster.local"
+  #
+  # JupyterHub
+  jupyterhub:
+    proxy:
+      secretToken: 11df35dab7527b3798de5224f06458db1e0526fa65700e81e18f17f89822e0cf
+    hub:
+      image:
+        name: "gitlab-registry.cern.ch/sciencebox/hotfixes/jupyterhub"
```

Comment:
Why do you need a special image? I would try to push everything to the real images

Reply:
Heritage. We are fixing this next week while aligning to the latest swan chart ;)